docs/command-reference/generic/script-list.md

```diff
@@ -0,0 +1,17 @@
+---
+description: List all scripts in the script cache.
+---
+
+# SCRIPT LIST
+
+## Syntax
+
+    SCRIPT LIST
+
+**Time complexity:** O(N) with N being the number of scripts in script cache.
+
+Returns information about all the scripts in the script cache.
+
+## Return
+
+[Array reply](https://redis.io/docs/reference/protocol-spec#resp-arrays) This command returns an array of elements. The first element is the SHA1 digest of the scripts added into the script cache. The second element is lua script.
```

Comment:
Currently we can return the third element - the post-processed script (sheband removed, auto-async applied). I added this at some point for debug reasons, but didn't remove it. But I think we shouldn't mention it here and else hide it behind as secret parameter